poetry.lock

```diff
@@ -1,10 +1,9 @@
-# This file is automatically @generated by Poetry and should not be changed by hand.
+# This file is automatically @generated by Poetry 1.6.0 and should not be changed by hand.
```

Comment:
Interesting, renovate reverted the change @iamogbz

Reply:
ah, I see. Renovate is using a newer version of poetry. I'll update and then see if I can specify the version with renovate

Reply:
https://github.com/tophat/syrupy/pull/793